svthalia/concrexit
# Make facedetection more noticeable #3427

Closes #3427.

feature: something new

Changes:
- added alert banners in the photos page depending on status of face detection. 
- added a link to 'Your photos' in user dropdown menu
- changed icons photos page. Search button now has icon instead of text. 'Your photos' button has a smiley face icon instead of user icon

How to test
1. Go to 'Photos' without any reference faces 
2. Click on 'face recognition' link in the alert banner
3. Upload reference faces and return to 'Photos'/Trigger different face detection events such as rejecting faces and check alert banner in 'Photos'
